File: scala-gatling/src/gatling/scala/org/openapitools/client/model/ResponseGeocoding.scala

package org.openapitools.client.model


case class ResponseGeocoding (
    _type: String,
    _features: List[ResponseGeocodingGeoJsonFeature]
)
object ResponseGeocoding {
    def toStringBody(var_type: Object, var_features: Object) =
        s"""
        | {
        | "type":$var_type,"features":$var_features
        | }
        """.stripMargin
}

File: fsharp-functions/OpenAPI/src/model/ResponseGeocoding.fs
namespace OpenAPI.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open OpenAPI.Model.ResponseGeocodingGeoJsonFeature

module ResponseGeocoding = 

  //#region ResponseGeocoding

  [<CLIMutable>]
  type ResponseGeocoding = {
    [<JsonProperty(PropertyName = "type")>]
    Type : string;
    [<JsonProperty(PropertyName = "features")>]
    Features : ResponseGeocodingGeoJsonFeature[];
  }
  
  //#endregion
  